src/main/scala/io/viash/Main.scala

```diff
@@ -323,13 +324,25 @@ object Main {
 
   def processConfigWithPlatform(
     config: Config, 
-    platformStr: Option[String]
+    platformStr: Option[String],
+    targetDir: Option[String]
   ): (Config, Option[Platform]) = {
     // add platformStr to the info object
     val conf1 = config.copy(
-      info = config.info.map{
-        _.copy(platform = platformStr)
-      }
+      info = config.info.map{_.copy(
```

Comment:
Should we store paths as relative paths to the project root?
Previously these were absolute paths (to verify what we do here now)

Reply:
I think relative paths would be best